homeassistant/components/zoneminder/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "flow_title": "ZoneMinder",
+    "step": {
+      "user": {
+        "title": "Add ZoneMinder Server.",
+        "data": {
+          "host": "Host and Port (ex 10.10.0.4:8010)",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "path": "ZM Path",
```

Comment:
Does this display the default values in the fields?

Reply:
This file only describes the text for the form. However, the defaults are shown.

![image](https://user-images.githubusercontent.com/1752370/93098656-f7ea3680-f65b-11ea-8b5f-e07600523777.png)

The username/password labels will be corrected once the translation service generate the correct en.json file and includes the values from the config_flow data. That won't happen until after merge.

Reply:
Got it, thanks. I was wanting to double check since the host key had the example here.